Stardew valley: Supported Mods documentation and changes required for legal reasons (#2038)
ArchipelagoMW/Archipelago: worlds/stardew_valley/docs/setup_en.md

- Archipelago from the [Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases)
    - (Only for the TextClient)
- Other Stardew Valley Mods [Nexus Mods](https://www.nexusmods.com/stardewvalley)

>     - There are [supported mods](https://github.com/agilbert1412/StardewArchipelago/blob/4.x.x/Documentation/Supported%20Mods.md), that you can add to your yaml and include them in the archipelago randomization

Reviewer: ```suggestion
    - There are [supported mods](https://github.com/agilbert1412/StardewArchipelago/blob/4.x.x/Documentation/Supported%20Mods.md) that you can add to your yaml to include them with the Archipelago randomization

```
